datafusion/core/tests/sqllogictests/README.md

```diff
@@ -23,7 +23,21 @@ This is the Datafusion implementation of [sqllogictest](https://www.sqlite.org/s
 
 #### Running tests
 
-`cargo test -p datafusion --test sqllogictests`
+```shell
+cargo test -p datafusion --test sqllogictests
+```
+
+Run tests with debug logging enabled:
+
+```shell
+RUST_LOG=debug cargo test -p datafusion --test sqllogictests
+```
+
+Run only the tests in `information_schema.slt`:
+
+```shell
+cargo test -p datafusion --test sqllogictests -- information_schema.slt
+```
```

Comment:
Could we also add a note on the substring matching ability?

Reply:
In https://github.com/apache/arrow-datafusion/pull/4497/commits/fd98463a02895c35e74f36ff1134e4f40cab8b20